pkg/tilesets/tilesets.go

```diff
@@ -141,3 +141,61 @@ func StandardTileSet() TileSet { //nolint:gocyclo // shallow loops for adding ti
 		Tiles:        tiles,
 	}
 }
+
+func OrderedMiniTileSet1() TileSet { //nolint:gocyclo
```

Comment:
This should be moved to a test that is using this.

Reply:
Moved